Documentation/Setup/Page/Index.rst

```diff
@@ -533,6 +533,10 @@ includeJS.[array]
                 # in the TS constants field.
                 file2 = javascript_uploaded_to_template*.js
                 file2.if.isTrue = {$myConstant}
+                
+                twitter = http://twitter.com/javascripts/blogger.js
```

Comment:
This does not exist. And if, it should use HTTPS and probably a fixed version to ensure the mentioned `integrity` matches.

Reply:
This was just moved from the doubled section I removed. Didn't check if this JS exists. What about using another JS like the latest of jQuery?

Reply:
Good idea. :-)

Reply:
Using jQuery now.